cpp/unreal_plugins/SimulationController/Source/SimulationController/CameraSensor.cpp

```diff
@@ -386,6 +386,31 @@ void CameraSensor::initializeSceneCaptureComponentNonFinalColor(USceneCaptureCom
     ASSERT(material);
     scene_capture_component->PostProcessSettings.AddBlendable(UMaterialInstanceDynamic::Create(material, scene_capture_component), 1.0f);
     scene_capture_component->ShowFlags.SetPostProcessMaterial(true);
+
+    // assign custom depth stencil values in segmentation mode by reading from our config system
+    if (render_pass_name == "segmentation") {
+
+        UWorld* world = parent_actor_->GetWorld();
+        std::map<std::string, int> semantic_tag_id_mapping = Config::get<std::map<std::string, int>>("SIMULATION_CONTROLLER.CAMERA_SENSOR.SEMANTIC_MAPPING");
+
+        for (TActorIterator<AActor> actor_itr(world); actor_itr; ++actor_itr) {
```

Comment:
`// TODO: factor this functionality into a general-purpose function for finding components by tag`

Reply:
Added